cognite/client/_api/diagrams.py

```diff
@@ -55,9 +56,13 @@ def _run_job(
         )
 
     @staticmethod
-    def _process_file_ids(ids: Union[Sequence[int], int, None], external_ids: Union[Sequence[str], str, None]) -> List:
+    def _process_file_ids(
+        ids: Union[Sequence[int], int, None],
+        external_ids: Union[Sequence[str], str, None],
+        file_references: Union[Sequence[FileReference], FileReference, None],
+    ) -> List:
         # Handle empty lists
-        if not external_ids and not ids:
+        if not (external_ids or ids or file_references):
             raise ValueError("No ids specified")
```

Comment:
Update error message to reflect no ids and no (...)

Reply:
This error also triggers if given `ids=0` or `external_ids=""`. Is this expected?

Reply:
Not expected. Ids, external ids and file references are all a type of file ids, but changing the error message anyway.

Reply:
Moving the check further down, to avoid the possibility of 0 or ""